packages/venia-ui/lib/components/Checkout/addressForm.js

```diff
@@ -28,16 +31,24 @@ const fields = [
 ];
 
 const AddressForm = props => {
-    const { countries, error, isSubmitting, onCancel, onSubmit } = props;
```

Comment:
Okay, this explains the snapshot discrepancy from earlier. Seems like this component no longer takes an `error` prop; the talon provides one instead.

Therefore I think we should:
  1. Remove `error` from this component's `propTypes`
  1. Fix up or remove the "renders validation block with message if address is incorrect" test in `addressForm.spec.js`. It's passing in an `error` prop right now. Perhaps we mock the talon's response instead.
  1. Update the snapshot again to ensure we're rendering an error message